### PR TITLE
feat(tooltip): creation of a new tooltip 

### PR DESCRIPTION
**Pull request by GusBaamonde**
component for report charts visits weekdays hours

- Add a mixin for the new tooltip
- Add a different animation to the new tooltip

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

new-tooltip

### CODE REVIEW

File: assets/scss/modules/_tooltip.scss [hunk 1]
```diff
@@ -116,3 +116,82 @@
     }
   }
 }
+
+/* New Tooltip II */
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_tooltip.scss]: Maybe can use a name more descriptive

File: assets/scss/modules/_tooltip.scss [hunk 2]
```diff
@@ -116,3 +116,82 @@
...
+    z-index: 5;
+    text-align: left;
+    border: 1px solid $dp-color-silver;
+    box-shadow: 0 0 0 4px rgba(204, 204, 204, 0.3);
+
+    &.bgColor {
+      background: rgba($dp-color-white, 0.95);
+    }
+
+    &:before {
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_tooltip.scss]: this is the arrow?

File: assets/scss/modules/_tooltip.scss [hunk 3]
```diff
@@ -116,3 +116,82 @@
...
+      height: $dp-spaces--lv0;
+      border: 5px solid transparent;
+      position: absolute;
+      bottom: $dp-spaces--lv0;
+      left: 50%;
+      transform: translate(-50%, 100%);
+      border-top: 5px solid rgba($dp-color-white, 0.95);
+    }
+
+    p {
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_tooltip.scss]: it's ok the styles of the content inside the tooltip are here? in that case always `p, span` has this styles

File: assets/templates/tooltip-component.html [hunk 4]
```diff
@@ -123,6 +123,27 @@ <h3>Tooltip</h3>
...
+              y horas de la pagina de reportes.
+            </p>
+            <hr class="dpsg-break" />
+            <div class="wrapper-tooltip">
+              <div class="dp-box">
+                <div class="dp-tooltip">
+                  <p class="wrap-tooltip dpsg-button-example">
+                    Tooltip II example
+                  </p>
+                  <div class="tooltip bgColor">
```
Comment (fcoronelmakingsense) [inline assets/templates/tooltip-component.html]: only need in webapp this css class? or need all structure?

**Comment by andresmoschini** [comment]
Nice! Does the tooltip work with CSS only? or the javascript was already there?

File: assets/scss/modules/_tooltip.scss (hunk 3 above)
Comment (GusBaamonde) [reply assets/scss/modules/_tooltip.scss]: For me they are fine, because these elements have particular characteristics of this component

**Comment by GusBaamonde** [comment]
> ¡Agradable! ¿La información sobre herramientas funciona solo con CSS? o el javascript ya estaba allí?

Only CSS

File: assets/scss/modules/_tooltip.scss (hunk 2 above)
Comment (GusBaamonde) [reply assets/scss/modules/_tooltip.scss]: Yes

File: assets/templates/tooltip-component.html (hunk 4 above)
Comment (GusBaamonde) [reply assets/templates/tooltip-component.html]: the structure is practically similar, only using the classes should it work

**Pull request merged**